cli/command/system/version.go

```diff
@@ -35,7 +36,11 @@ Client:{{if ne .Platform.Name ""}} {{.Platform.Name}}{{end}}
  OS/Arch:	{{.Os}}/{{.Arch}}
  Context:	{{.Context}}
  Experimental:	{{.Experimental}}
-{{- end}}
+ {{- range $backend := .Backends}}
+ {{$backend.Name}}:
```

Comment:
question: Should we add a sub section title `Backends` ?

Reply:
Good question, currently this looks like: 
```
$ ./build/docker-darwin-amd64 version
Client:
 Version:           20.10.0-dev
 API version:       1.41
 Go version:        go1.13.15
 Git commit:        73cbd63ff
 Built:             Fri Jan  8 16:34:46 2021
 OS/Arch:           darwin/amd64
 Context:           default
 Experimental:      true
 Cloud integration:
  Version:          v1.0.6-16
```
Ideally I would prefer to display a single line for each backend and version, but the backend name is unknown in the template, making it hard to align the 2nd column. Also, we could have more info later on about backends (supported context type, ...).